src/website/utils/validators.py

```
@@ -0,0 +1,61 @@
+        return
+
+    allowed_extensions = ['pdf', 'doc', 'docx',
+                          'xls', 'xlsx', 'ppt', 'pptx', 'txt']
+    extension_validator = FileExtensionValidator(allowed_extensions)
+
+    try:
+        extension_validator(file)
+    except ValidationError:
+        raise ValidationError(
+            f"Unsupported file extension. Allowed extensions are: {', '.join(allowed_extensions)}."
+        )
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)